data/json/items/comestibles/junkfood.json

```diff
@@ -71,7 +71,7 @@
     "type": "COMESTIBLE",
     "id": "toasterpastry",
     "name": { "str": "toaster pastry", "str_pl": "toaster pastries" },
-    "copy-from": "toasterpastryfrozen",
+    "copy-from": "toastem4",
```

Comment:
Why change the copy from?

Reply:
This one is an accident, probably a legacy of a previous draft at this PR when I was going to rework ingredients too. Reverting it.